CHANGELOG.md

```diff
@@ -6,8 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [v1.3.0] - 2020-02-10
 
+### Breaking
```

Comment:
Should we all caps and change to BREAKING CHANGES for consistency with the main repo?